tools/keepalived/keepalived/check/libipvs.c

```diff
@@ -243,6 +243,8 @@ int ipvs_update_service_by_options(ipvs_service_t *svc, unsigned int options)
 	}
 
 	if (options & OPT_HASHTAG) {
+		app.user.flags &= ~ IP_VS_SVC_F_QID_HASH;
+		app.user.flags &= ~ IP_VS_SVC_F_SIP_HASH;
```

Comment:
Line 209~210 cleared these two option flags. Was it not working?

Reply:
> Line 209~210 cleared these two option flags. Was it not working?

The purpose on 209 and 210 is to clear the sip and qid flag  configuration obtained from dpvs when the scheduling algorithm is not conhash